### PR TITLE
💄 Give a proper code styling to all PHP classes

### PR DESCRIPTION
**Pull request by guzzilar**
#### 1. Objective

Give a proper code styling to all PHP classes in Omise-Magento project to improve readability to the code.

**Related information**:
Related issue(s): 🙅
Related ticket(s): 🙅

#### 2. Description of change

Rules that I used,

  1. **comment block, have 1 break line in every annotations.**
      _Bad_
      ```php
      /**
       * Description
       * @param string $a
       * @param int $b
       * @return \OmiseObject 
       */
      ```
      _Good_
      ```php
      /**
       * Description
       *
       * @param  string $a
       * @param  int    $b
       *
       * @return \OmiseObject 
      ```

  2. **1 space between `!condition`.**
      _Bad_
      ```php
      if (!$paid) {
          // ...
      } 
      ```  
      _Good_
      ```php
      if (! $paid) {
          // ...
      } 
      ```

  3. **No shorthand syntax for conditional block.**
      _Bad_
      ```php
      if (! $paid)
          echo "failed";
      ```
      _Good_
      ```php
      if (! $paid) {
          echo "failed";
      }
      ```
  4. **1 breakline at the end of a file.**

  5. **Formatting code vertically alignment**
      _Normal_
      ```php
      $person.FirstName = "Chris";                
      $person.Surname = "McGrath";                
      $person.Age = 24;                           
      $person.Occupation = "Software Developer";  
      $person.HomeTown = "Brisbane";              
      ```
      _Good_
      ```php
      $person.FirstName  = "Chris"; 
      $person.Surname    = "McGrath"; 
      $person.Age        = 24; 
      $person.Occupation = "Software Developer"; 
      $person.HomeTown   = "Brisbane";
      ```
      _note, example from http://www.codealignment.com/Details.html_

  6. **1 space for variable string concatenation**
      _Bad_
      ```php
      $name     = 'Guzzilar';
      $sentence = 'Have a nice day!';

      echo "Hello, ".$name.". ".$sentence;
      // Hello, Guzzilar. Have a nice day!
      ```
      _Good_
      ```php
      $name     = 'Guzzilar';
      $sentence = 'Have a nice day!';

      echo "Hello, " . $name . ". " . $sentence;
      // Hello, Guzzilar. Have a nice day!
      ```

#### 3. Quality assurance

No test, this changes are not affect to functions

#### 4. Impact of the change

No.

#### 5. Priority of change

Normal

#### 6. Additional Notes

- For PHP CodeSniffer result, please check the screenshot below.
  screen shot 2560-01-27 at 5 05 24 pm

  *note, there still has some `exceeds 120 chars` left, I intended to leave it like that to avoid code conflict cuz PR #41 is also touching those files.

### CODE REVIEW

**Comment by nimid** [comment]
@guzzilar I found Magento 1 has not been defined the PHP coding standard officially.

So, can PHP coding standard be some rules from [PSR-2](http://www.php-fig.org/psr/psr-2/)?

**Comment by guzzilar** [comment]
@nimid https://github.com/omise/omise-magento/pull/40#issuecomment-275601261
sure.

**Comment by nimid** [comment]
@guzzilar I have run a command `phpcs --standard=PSR2 --ignore=*/omise-php/* omise-magento/src/`. 

The below is some part of report that I have cut and remain only conform with the pull request description.

**1. FILE: ...e-magento/src/app/code/community/Omise/Gateway/Block/Form/Cc.php**
```
  37 | ERROR   | [x] Inline control structures are not allowed
  39 | ERROR   | [x] Expected 1 space after ELSE keyword; newline
     |         |     found
  39 | ERROR   | [x] Inline control structures are not allowed
  54 | ERROR   | [x] Expected 1 space before "=>"; 0 found
  54 | ERROR   | [x] Expected 1 space after "=>"; 0 found
 102 | ERROR   | [x] Expected 1 space after IF keyword; 0 found
 102 | ERROR   | [x] Expected 1 space after closing parenthesis;
     |         |     found 0
 128 | ERROR   | [x] Line indented incorrectly; expected 4 spaces,
     |         |     found 5
 129 | ERROR   | [x] Opening brace indented incorrectly; expected 5
     |         |     spaces, found 4
 139 | ERROR   | [x] Closing brace indented incorrectly; expected 5
     |         |     spaces, found 4
```

**2. FILE: ...ommunity/Omise/Gateway/controllers/Adminhtml/OmiseController.php**
```
92 | ERROR   | [x] Inline control structures are not allowed
```

**3. FILE: .../Omise/Gateway/data/omise_gateway_setup/data-install-0.0.0.1.php**
```
 17 | ERROR | [x] Expected 1 newline at end of file; 0 found
```


**4. FILE: ...ise-magento/src/app/code/community/Omise/Gateway/Helper/Data.php**
```
 5 | ERROR | [x] Expected 1 newline at end of file; 0 found
```

**5. FILE: ...ise-magento/src/app/code/community/Omise/Gateway/Model/Omise.php**
```
  3 | ERROR   | [x] Expected 1 space after IF keyword; 0 found
  8 | ERROR   | [x] Expected 1 space after IF keyword; 0 found
```

**6. FILE: ...nto/src/app/code/community/Omise/Gateway/Model/PaymentMethod.php**
```
 121 | ERROR   | [x] Inline control structures are not allowed
 127 | ERROR   | [x] Inline control structures are not allowed
```

**7. FILE: ...munity/Omise/Gateway/sql/omise_gateway_setup/install-0.0.0.1.php**
```
 23 | ERROR | [x] Expected 1 newline at end of file; 0 found
```

**Comment by guzzilar** [comment]
<3 that's really helpful @nimid 
I should do the same.

**Comment by guzzilar** [comment]
@nimid can I have the PHPCode_Sniffer version you're using?
Wanna have it in the repo (add to composer.json)

File: src/app/code/community/Omise/Gateway/Model/OmiseCharge.php [hunk 1]
```diff
@@ -17,7 +19,9 @@ public function createOmiseCharge($params)
 
     /**
      * Capture a charge that retrieve from charge id
-     * @param array $params
+     *
+     * @param  array $params
```
Comment (nimid) [inline src/app/code/community/Omise/Gateway/Model/OmiseCharge.php]: @guzzilar The difference of parameter name in PHPDoc and source code.

PHPDoc is `$params`

Source code is `$id`

**Comment by nimid** [comment]
@guzzilar https://github.com/omise/omise-magento/pull/40#issuecomment-275610365

I using `PHP_CodeSniffer version 2.7.0 (stable) by Squiz (http://www.squiz.net)`.

**Comment by guzzilar** [comment]
@nimid https://github.com/omise/omise-magento/pull/40#issuecomment-275611038  👍

**Comment by guzzilar** [comment]
@nimid could you review this PR, https://github.com/omise/omise-magento/pull/45

To import CodeSniffer to the project.

**Comment by guzzilar** [comment]
rebased with `1-stable`

**Comment by guzzilar** [comment]
@nimid Done for update code

For the result of PHP CodeSniffer, please check the screenshot below.

screen shot 2560-01-27 at 5 05 24 pm

*note, there still has some `exceeds 120 chars` left, I intended to leave it like that to avoid code conflict cuz PR #41 is also touching those files.

**Comment by guzzilar** [comment]
also, updated PR description.

File: src/app/code/community/Omise/Gateway/Model/OmiseCharge.php (hunk 1 above)
Comment (guzzilar) [inline src/app/code/community/Omise/Gateway/Model/OmiseCharge.php]: Updated in https://github.com/omise/omise-magento/pull/40/commits/ab6bde3a3a5308ed1e636e81546348dc2472d8ba 👍

**Comment by guzzilar** [comment]
@nimid 
For comments
- [pullrequestreview-18948230](https://github.com/omise/omise-magento/pull/40#pullrequestreview-18948230)
- [pullrequestreview-18949778](https://github.com/omise/omise-magento/pull/40#pullrequestreview-18949778)
- [pullrequestreview-18950688](https://github.com/omise/omise-magento/pull/40#pullrequestreview-18950688)
- [pullrequestreview-18951194](https://github.com/omise/omise-magento/pull/40#pullrequestreview-18951194)
- [pullrequestreview-18951509](https://github.com/omise/omise-magento/pull/40#pullrequestreview-18951509)
- [pullrequestreview-18953449](https://github.com/omise/omise-magento/pull/40#pullrequestreview-18953449)

Done here 👍 
https://github.com/omise/omise-magento/pull/40/commits/0defae6acade7fe8074407f446e3c1e4dc23d00a

**Comment by nimid** [comment]
@guzzilar Apart from [a requested change](https://github.com/omise/omise-magento/pull/40#pullrequestreview-19530164), there are no any part of code that I found it break the rules in the pull request.

So, this pull request has been approved with the conclusion below.
- The review by human is hard and take time. It can be missed. The tool, PHP_CodeSniffer, can help to speed up.
- Although the PSR-2 has the publicly accepted rules, but in the pull request description, the rules, 1, 2, 5 and 6 are the extended rules from the PSR-2.
- With the extended rules, the default of PHP_CodeSniffer can not help to check.
- [Some PSR-2 rules have been excluded](https://github.com/omise/omise-magento/blob/1-code-styling/ruleset.xml).

**Pull request merged**